pkg/frontend/querymiddleware/model_extra_test.go

```diff
@@ -134,12 +134,10 @@ func TestMetricQueryRequestCloneHeaders(t *testing.T) {
 				}
 
 				r := originalReq.WithHeaders(newHeaders)
-				require.NoError(t, err)
```

Comment:
there's one more stray check on line 126

Reply:
I missed it, fixed